logprep/util/configuration.py

```diff
@@ -103,7 +119,11 @@ def create_from_yaml(path: str) -> "Configuration":
         try:
             configuration = config_getter.get_json()
         except ValueError:
-            configuration = config_getter.get_yaml()
+            try:
+                configuration = config_getter.get_yaml()
+            except ScannerError as error:
+                print_fcolor(Fore.RED, f"Error parsing YAML file: {path}\n{error}")
+                sys.exit(1)
```

Comment:
Can you also add a test case for this except block please? So far I don't think that it is tested.